tests/Endpoints/SessionTest.php

```diff
@@ -76,6 +74,9 @@ public function putsToTest()
             'postrequisite' => ['postrequisite', 2],
             'emptyPostrequisite' => ['postrequisite', null],
             'prerequisites' => ['prerequisites', [2]],
+            'description' => ['description', $this->getFaker()->text],
+            'blankDescription' => ['description', ''],
+            // 'nullDescription' => ['description', null], // currently broken [ST 2020/07/24]
```

Comment:
Should pass `, $skipped = true` instead of commenting this out. I'm hesitant to approve though if this doesn't work.

Reply:
Ah, I see the comment on the purifier and understand why this is broken now. Will approve once skipped instead of commented.

Reply:
you got it.